2017-09-12 15:50:44,516 - remote_connection.py[line:478] - DEBUG: POST http://127.0.0.1:4723/wd/hub/session {"capabilities": {"alwaysMatch": {"platformName": "Android"}, "firstMatch": [{}]}, "desiredCapabilities": {"deviceName": "OPPO R7", "unicodeKeyboard": true, "app": "H:\\project\\Nidone_appium\\app-client-debug-test-2.3.13.apk", "resetKeyboard": true, "platformVersion": "4.4.4", "platformName": "Android"}}
2017-09-12 15:50:44,532 - remote_connection.py[line:565] - DEBUG: Finished Request
2017-09-12 15:50:45,548 - remote_connection.py[line:478] - DEBUG: POST http://127.0.0.1:4723/wd/hub/session {"capabilities": {"alwaysMatch": {"platformName": "Android"}, "firstMatch": [{}]}, "desiredCapabilities": {"deviceName": "OPPO R7", "unicodeKeyboard": true, "app": "H:\\project\\Nidone_appium\\app-client-debug-test-2.3.13.apk", "resetKeyboard": true, "platformVersion": "4.4.4", "platformName": "Android"}}
2017-09-12 15:50:45,548 - remote_connection.py[line:565] - DEBUG: Finished Request
2017-09-12 15:50:46,548 - remote_connection.py[line:478] - DEBUG: POST http://127.0.0.1:4723/wd/hub/session {"capabilities": {"alwaysMatch": {"platformName": "Android"}, "firstMatch": [{}]}, "desiredCapabilities": {"deviceName": "OPPO R7", "unicodeKeyboard": true, "app": "H:\\project\\Nidone_appium\\app-client-debug-test-2.3.13.apk", "resetKeyboard": true, "platformVersion": "4.4.4", "platformName": "Android"}}
2017-09-12 15:51:01,671 - remote_connection.py[line:565] - DEBUG: Finished Request
2017-09-12 15:51:01,671 - log.py[line:69] - INFO:   run 登录测试3.
2017-09-12 15:51:16,683 - remote_connection.py[line:478] - DEBUG: POST http://127.0.0.1:4723/wd/hub/session/dba1f4fe-e154-4ea1-972a-8fe4623edc91/context {"sessionId": "dba1f4fe-e154-4ea1-972a-8fe4623edc91", "name": "WEBVIEW"}
2017-09-12 15:51:16,683 - remote_connection.py[line:565] - DEBUG: Finished Request
2017-09-12 15:51:16,683 - Test_Case.py[line:84] - ERROR: Message: 

2017-09-12 15:51:16,683 - mkdir_log_directory.py[line:29] - INFO: H:\project\Nidone_appium\log\2017-09-12_15_50_43_error目录创建成功
2017-09-12 15:51:18,433 - log.py[line:48] - INFO: 错误截图已保存在 H:\project\Nidone_appium\log\2017-09-12_15_50_43_error
2017-09-12 15:51:18,433 - log.py[line:63] - INFO: 记录错误日志
2017-09-12 15:51:18,433 - log.py[line:71] - INFO:   登录测试3 end.
2017-09-12 15:51:18,433 - remote_connection.py[line:478] - DEBUG: DELETE http://127.0.0.1:4723/wd/hub/session/dba1f4fe-e154-4ea1-972a-8fe4623edc91 {"sessionId": "dba1f4fe-e154-4ea1-972a-8fe4623edc91"}
2017-09-12 15:51:18,433 - remote_connection.py[line:565] - DEBUG: Finished Request
